### PR TITLE
Added Chinese translations

### PR DESCRIPTION
**Pull request by Lien-Chu**

### CODE REVIEW

File: src/i18n/translation.json [hunk 1]
```diff
@@ -32,5 +32,23 @@
...
+      "Advanced": "困難",
+      "Pages": "頁數",
+      "Included Operators": "運算符號",
+      "Name": "名字",
+      "Date": "日期",
+      "Randomize": "隨機",
+      "Print": "列印"
+    }
   }
+
```
Comment (dkaksl) [inline src/i18n/translation.json]: Why is there a blank line here?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (dkaksl) [inline .vscode/settings.json]: What is this file for?

File: src/i18n/translation.json (hunk 1 above)
Comment (Lien-Chu) [reply src/i18n/translation.json]: Fixed

File: .vscode/settings.json (hunk 2 above)
Comment (Lien-Chu) [reply .vscode/settings.json]: Fixed

**Pull request merged**